### ISSUE TITLE
Accidental Change on recent PR

### ISSUE THREAD

**Issue by davama** [ytti/oxidized#1870]
Hello @wk 

Recently PR 52e9d8ae2eee7878a1130c316be6926adc45436f was merged but with it was a change that I wanted to separately submit a PR for but it got merged together... that's my fault.

The change is here https://github.com/ytti/oxidized/commit/11c7cfccda8451860933d0cc18edf20541948bc7#diff-9ce4dc0fff22a9c6d977ea31eb6172ba

As you can see, it addresses issue here: https://github.com/ytti/oxidized/issues/1254

Not sure what steps are needed to adjust/correct this.

Please let me know.

Thank you,
Dave

**Comment by wk** [comment]
Hi, 

Thanks for the heads-up! Since I merged #1820, the slip-up is all mine; an unrelated change to a different model should've never passed review.

I've introduced #1871 to revert this change; if you want it in, please open a separate PR (with changelog entry, etc.) that tackles this issue alone after #1871 is merged.

**Comment by davama** [comment]
Thank you @wk
